sematic/ui/packages/common/src/pages/RunSearch/RunList.tsx

```diff
@@ -52,50 +58,37 @@ const NameSection = styled.div`
     position: relative;
 `;
 
-const StyledRunReferenceLink = styled(RunReferenceLink)`
+const StyledRunReferenceLink = styled(RunReference)`
     color: ${theme.palette.mediumGrey.main};
 `;
 
-const RunData = {
-    ID: '3es92wd',
-    created: '2021-10-01T12:00:00Z',
-    name: 'MNIST PyTorch Example',
-    importPath: 'examples.mnist.pipeline',
-    tags: ['example', 'pytorch', 'mnist', 'demo'],
-    owner: 'Demo User',
-    status: 'RUNNING',
-    failedAt: undefined as undefined | string,
-    resolvedAt: undefined as undefined | string,
-    canceledAt: undefined as undefined | string,
-}
-
-const columnHelper = createColumnHelper<typeof RunData>()
+const columnHelper = createColumnHelper<Run>()
 
 const columns = [
-    columnHelper.accessor('ID', {
+    columnHelper.accessor('id', {
         meta: {
             columnStyles: {
                 width: "5.923%",
             }
         },
         header: 'ID',
-        cell: info => <StyledRunReferenceLink variant={'inherit'} runId={info.getValue()} />,
+        cell: info => <StyledRunReferenceLink runId={info.getValue()} />,
     }),
-    columnHelper.accessor('created', {
+    columnHelper.accessor('created_at', {
         meta: {
             columnStyles: {
                 width: "12.3396%",
-                minWidth: "140px"
+                minWidth: "120px"
             }
         },
         header: 'Submitted at',
         cell: info => DateTimeLongConcise(parseJSON(info.getValue())),
     }),
-    columnHelper.accessor(data => [data.name, data.importPath], {
+    columnHelper.accessor(data => [data.name, data.function_path], {
```

Comment:
When resizing, `Name` and only `Name` is being compressed, while other columns are kept wide. `Name` and `Tags` get overlapped. Please compress all columns and fix the overlapping.

![image](https://github.com/sematic-ai/sematic/assets/1894533/f0f7a254-bb8b-423b-b6b3-64d88c0d2277)

Reply:
The `submitted at` shouldn't have a line break, and it will be fixed. 

After that, the row shouldn't wrap lines, otherwise, it cannot maintain a fixed height of 50px, which loses the beauty of the design (this is however controversial).

I never expected the table width to reduce this narrow. Therefore, I suggest we have a minimum width of the table, and show a horizontal scrollbar if the browser window is that narrow. How does that sound?